code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -497,13 +537,20 @@
 		"creme_de_cacao",
 		"triple_sec",
 		"sake"
-	)
+	)//prevents the booze machine from obtaining chemical upgrades.
+	upgrade_reagents = null
+	upgrade_reagents2 = null
+	upgrade_reagents3 = null
 	emagged_reagents = list(
 		"ethanol",
 		"iron",
+		"clownstears",
 		"minttoxin",
 		"atomicbomb",
-		"fernet"
+		"fernet",
+		"slimejelly",
```

Comment:
no, it's slime jelly for a reason, and just being able to machine out this is ridiculous when it does such high damage.